float64 pos_x
float64 pos_y
float64 pos_z
int64 giro_x
int64 giro_y
int64 giro_z
float64 vel
float64 ace

